migrate/20240423_add_fraud_columns.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    alter_table(:payment_method) do
+      add_column :card_fingerprint, :text, null: true
+      add_column :fraud, :boolean, default: false, null: false
+    end
+  end
+
+  down do
+    alter_table(:payment_method) do
+      drop_column :card_fingerprint
+      drop_column :fraud
+    end
+  end
```

Comment:
nit: `change` automatically handles down
```suggestion
  change do
    alter_table(:payment_method) do
      add_column :card_fingerprint, :text, null: true
      add_column :fraud, :boolean, default: false, null: false
    end
  end
```